api_client/python/timesketch_api_client/client.py

```diff
@@ -247,7 +248,7 @@ def _create_oauth_session(
             flow.redirect_uri = self.DEFAULT_OAUTH_OOB_URL
 
         if run_server:
-            _ = flow.run_local_server()
+            _ = flow.run_local_server(host="localhost", port=8080, open_browser=False)
```

Comment:
Please add host, port, open_browser as parameter to this function, with these values as defaults. Then anyone using the api client can choose.

Reply:
Done